system-int-test/src/cmr/system_int_test/utils/subscription_util.clj

```diff
@@ -110,18 +111,34 @@
   ([params options]
    (search/find-refs :subscription params options)))
 
+(defn- validate-subscription-response-format
+  [subscription]
+  ;; verifying schema keys are included
+  (is (= #{:collection_concept_id
+           :concept_id
+           :name
+           :native_id
+           :provider_id
+           :revision_id
+           :subscriber_id}
+         (set (keys subscription))))
+  ;; verifying no dash in keys
+  (is (not-any? #(string/includes? % "-") (map name (keys subscription)))))
```

Comment:
If we have the stricter check on the line above, we don't really need this line here. My original suggestion is to only have this check, not he check above. That way, we don't need to make code change when say in the future, we add `additional_attribute` in the subscription response.

Reply:
I want you to delete line 117.